core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -73,6 +90,16 @@ oppia.directive('collectionNodeCreator', [function() {
           );
         };
 
+        var cleanExplorationId = function(newExplorationId) {
```

Comment:
I'm not sure if using slice method will be a good idea here because it performs the slicing as per the index. Like, we have some demo explorations with id's like 5 and 0 and also, on the live project -- the id's are of 12 characters. Let's wait for @seanlip on this.